### ISSUE TITLE
Release 30

### ISSUE THREAD

**Issue by gundelsby** [nrkno/Sofie-TV-automation#29]
## Development status

The development started mid January 2021. Testing started 2020-02-08. Released 2021-03-22.

- [X] Planning
- [X] Working on
- [x] Features frozen, in testing.  
- [x] Tests complete, update docs
- [x] Released

## Summary of features

* Translations for blueprint messages  ([c957ec8](https://github.com/nrkno/tv-automation-server-core/commit/c957ec87cba4ac18e59aa2eec13b3f92d9af673d))
* Prompter: adds support for multiple midi inputs ([c68d74c](https://github.com/nrkno/tv-automation-server-core/commit/c68d74ca06fd8dbe4c5545d651b76a9a8973769d))
*  PartInstances without Parts ([#417](https://github.com/nrkno/tv-automation-server-core/issues/417)) ([8895258](https://github.com/nrkno/tv-automation-server-core/commit/889525886f986f69fb91af55b49ed9b93780b67c))
* Layer Mappings defined by manifest ([63eb7b6](https://github.com/nrkno/tv-automation-server-core/commit/63eb7b6fe20c4768f584975054261a6bc34792e5))
* Building with github actions, including sonarcloud ([5241061](https://github.com/nrkno/tv-automation-server-core/commit/52410614519234a31658ad7410b1a43e3772a739))
* Double click on AdLib in the Dashboard and Buckets to trigger ([#430](https://github.com/nrkno/tv-automation-server-core/issues/430)) ([8d4f75c](https://github.com/nrkno/tv-automation-server-core/commit/8d4f75cbfceafbadfca672c255b0193d0cd68342))
* Support for looping rundowns (e.g.  [82b0ab0](https://github.com/nrkno/tv-automation-server-core/commit/82b0ab0ec259befcba4406b2ac2b752fb932d50e))
* Activation pop-up when doing a take on an inactive rundown ([6dd6412](https://github.com/nrkno/tv-automation-server-core/commit/6dd64124b7e1d7ef961e0c265317a7834aa988ef))
* Some reductions in the number of Unsync messages shown during shows with rundown playlists.
* Better handling of Quantel failover between ISA Managers.

## Expected versions of components:
| Component | Expected version |
| -- | -- |
| [Core](https://bit.ly/2yFZA9X) | 1.18 |
| [Blueprints API ( Core )](https://bit.ly/2YtWk0L) | 1.18 |
| [Blueprints API ( TSR )](https://bit.ly/2GTAmcF) | 5.5.5 |
| [Playout Gateway](https://bit.ly/2OKGbzn) | 1.16 |
| [Mos Gateway](https://bit.ly/2Tc1wR1) | 1.9.1 |
| [Media Manager](https://bit.ly/2ZDRjiD) | 1.5 |
| [Quantel Gateway](https://github.com/nrkno/tv-automation-quantel-gateway) | 1.1.0 |

**Comment by brandonkal** [comment]
Hello. I am trying to wrap my head around the release process. This is listed as the current stable release but:

1. The commits on the release branch failed the CI branch and they failed when merged into master.
2. It appears that all the libraries have failing tests on npm published releases.
3. This appears to be common across releases.
4. So my question is why are these being released as stable with failing tests? If the test suite is ignored shouldn't it be deleted? I must assume there are some undocumented out-of-band tests but is nrko really using release 30?

**Comment by mint-dewit** [comment]
Hi @brandonkal, I can confirm that NRK is really using release 30. Our CI tests typically need to complete successfully before we can do an automated release but I believe there are some exceptions. Gateways and server-core for example are released by Jenkins and a human needs to vet the release there. Our release process is quite lengthy and many steps make it easy to miss one unfortunately. This is something we look to improve continuously and I'll bring it up with the team to see if we can do an audit on this soon.

**Comment by jstarpl** [comment]
Release superseded by #31.
